packages/server/src/api/app/bsky/getLikedBy.ts

```diff
@@ -48,6 +48,8 @@ export default function (server: Server) {
         encoding: 'application/json',
         body: {
           uri,
+          cid,
```

Comment:
I saw that `cid` is in the output per the schema but was never being used, so I added it here.  If this wasn't the intent, just give a shout!